server/routes/checkYourAnswers/index.ts

```diff
@@ -30,8 +45,8 @@ export default function CheckAnswersRoutes({
   const get = (path: string, handler: RequestHandler) => router.get(path, asyncMiddleware(handler))
   const post = (path: string, handler: RequestHandler) => router.post(path, asyncMiddleware(handler))
 
-  get('/:prisonerNumber/:id', checkYourAnswers.view)
-  post('/:prisonerNumber/:id', checkYourAnswers.submit)
+  get('/:prisonerNumber/:adjudicationNumber', checkYourAnswers.view)
+  post('/:prisonerNumber/:adjudicationNumber', checkYourAnswers.submit)
   get('/:prisonerNumber/:id/report', checkYourAnswersBeforeChangeReporter.view)
```

Comment:
I will change the rest of these to use "adjudicationNumber" rather than id in the next tickets